windows-nsis/build-snapshot

```diff
@@ -12,6 +12,7 @@ get_full_path() {
 }
 
 GIT="${GIT:-git}"
+OPENVPN_VERSION="${OPENVPN_VERSION:-2.3_git}"; export OPENVPN_VERSION
```

Comment:
it is bash variable otherwise, it does not work. we need environment variable here